samples/benchmark/pom.xml

```diff
@@ -17,6 +17,10 @@
 
```

Comment:
Ignore the following files. I guess I messed up when using Git. I'll fix it during next iteration. I am sorry.